calebhsu/CoMake
# Service error handling

Closes #222 

Handles service errors in front end and displays loading text when sharing

To verify:
- run npm install for website

- modify ServiceEndpoints.jsx to point to a fake endpoint.

- UserInfo - log out, delete your user account in firebase, try to log in. Should open an alert window (couldn't do a snackbar, got complex with the way we are loading landing/dash based on auth)

- Create - log in with an existing user (so UserInfo doesn't break) and then try and create canvas. Should open red snackbar.

- Sharing - try to share, error text should display message about being unable to share
